Clients/src/presentation/pages/FileManager/index.tsx

```
@@ -0,0 +1,223 @@
+              sortDirection === "asc" && sortField === "uploader"
+                ? AscendingIcon
+                : DescendingIcon
+            }
+            alt="Sort"
+            sx={{ width: 16, height: 16, ml: 0.5 }}
+          />
+        </Box>
+      ),
+    },
+    { id: 5, name: "Action" },
+  ];
```

Biome

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)